CHANGELOG.md

```diff
@@ -4,6 +4,26 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 5.14.0
+### Added
+* Service: Added error message to catch VMSS Model `HttpResponseError` [#2384](https://github.com/microsoft/onefuzz/pull/2384)
```

Comment:
Don't need to include. Just error reporting.

Reply:
Removed